contracts/utils/ICS20Lib.sol

```diff
@@ -153,42 +144,40 @@ library ICS20Lib {
     }
 
     /// @notice getDenomPrefix returns an ibc path prefix
-    /// @param port Port
-    /// @param client client
+    /// @param portId Port
+    /// @param clientId client
     /// @return Denom prefix
-    function getDenomPrefix(string calldata port, string calldata client) internal pure returns (bytes memory) {
-        return abi.encodePacked(port, "/", client, "/");
+    function getDenomPrefix(string calldata portId, string calldata clientId) internal pure returns (bytes memory) {
+        return abi.encodePacked(portId, "/", clientId, "/");
+    }
+
+    /// @notice removeHop removes the first hop from the denom trace.
+    /// @param denom Denom to remove the hop from
+    /// @param hop Hop to remove (it must be the first hop)
+    /// @return The new denom with the first hop removed
+    function removeHop(bytes memory denom, bytes memory hop) internal pure returns (bytes memory) {
+        return Bytes.slice(denom, hop.length);
     }
```

Comment:
I wonder what the efficiency changes would be if you inlined this code instead. We may wanna consider it if it has substantial efficiency. Since function code is also easy to read, you could simply inline this and add a comment inline.